exporter/prometheus/prometheus_test.go

```diff
@@ -187,7 +55,7 @@ func TestMetricsEndpointOutput(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to create prometheus exporter: %v", err)
 	}
-	view.RegisterExporter(exporter)
+	//view.RegisterExporter(exporter)
```

Comment:
Nit: if these are not needed any more then just remove them.

Reply:
fixed.